charts/qdrant/templates/statefulset.yaml

```diff
@@ -284,7 +284,7 @@ spec:
         name: {{ .Values.snapshotPersistence.snapshotsVolumeName | default "qdrant-snapshots" }}
         labels:
           app: {{ template "qdrant.name" . }}
-          {{- with .Values.snapshotPersistence.labels }}
+          {{- with .Values.snapshotPersistence.additionalLabels }}
```

Comment:
Should we care about backward compatibility? If any user already using `labels`, their release will break after update. Maybe we can merge `labels` and `additionalLabels` here.